components/stacks-network/Cargo.toml

```diff
@@ -25,15 +24,16 @@ reqwest = { version = "0.11", default-features = false, features = [
     "json",
     "rustls-tls",
 ] }
-tui = { version = "0.18.0", default-features = false, features = ["crossterm"] }
 crossbeam-channel = "0.5.6"
-
+crossterm = { version = "0.27.0" }
+ratatui = { version = "0.25.0", default-features = false, features = ["crossterm"] }
 chrono = "0.4.31"
 futures = "0.3.12"
 base58 = "0.2.0"
+tokio = { version = "1.35.1", features = ["full"] }
 
-# chainhook-sdk = { version = "=0.11", default-features = true, path = "../../../chainhook/components/chainhook-sdk" }
-chainhook-sdk = { version = "=0.11", default-features = true }
+chainhook-sdk = { default-features = true, git = "https://github.com/hirosystems/chainhook.git", rev = "a938e61846a430750ad31d84b2032efabf6d039f"}
```

Comment:
Same comment about tracking these in an issue

Reply:
https://github.com/hirosystems/clarinet/issues/1313